dvc/config.py

```diff
@@ -224,6 +224,8 @@ class RelPath(str):
         "row_limit": All(Coerce(int), Range(1)),
         "row_cleanup_quota": All(Coerce(int), Range(0, 100)),
     },
+    # section for experimental features
+    "feature": {Optional("parametrization", default=False): Bool},
```

Comment:
Maybe `experimental` after all? Just to emphasize that this is for experimental features. `feature` could potentially be used for non-experimental but opt-in features.

Reply:
Is there any difference between the opt-in feature and experimental feature? In theory, yes, but does not seem to be any different in practice.

Reply:
@skshetry There is a difference in how people will understand it. At least in my view, `feature` is expected to contain well-off stable or almost stable opt-in functionality, while `experimental`(aka `experimental feature`) has the unstable nature of the section in the name itself, so you really know what you are getting into.

Reply:
In that case, "opt-in" features are those that are documented and experimental features are not documented.